src/model/model.ts

```diff
@@ -36,9 +43,24 @@ export interface NestedFields {
 
 export interface Model<Fields>
   extends Omit<RawModel, 'fields' | 'indexes' | 'triggers' | 'presets'> {
+  /**
+   * The fields that make up this model's schema.
+   */
   fields?: Fields;
+
+  /**
+   * Predefined queries that can be reused across the application.
```

Comment:
```suggestion
   * Predefined query instructions that can be reused across multiple different queries.
```

The presets are re-used by defining a query that uses the preset.